go.mod

```diff
@@ -1,25 +1,30 @@
 module github.com/confluentinc/ccloud-sdk-go-v1-public
 
 require (
-	github.com/confluentinc/proto-go-setter v0.0.0-20201026155413-c6ceb267ee65
-	github.com/dghubble/sling v1.2.1-0.20181125223409-7458fd7fa70b
-	github.com/envoyproxy/protoc-gen-validate v0.6.7
+	github.com/confluentinc/cc-structs/operator v0.1543.0
```

Comment:
We shouldn't be adding private repositories

Reply:
it was in tools.go. removed it!